doc/content/htgr/generic-pbr-tutorial/step1.md

```diff
@@ -0,0 +1,180 @@
+
+Then we use the `GeneratedMeshGenerator` to create a Cartesian mesh
+of the desired height and width. Note, how we use the ${x} syntax, where x
+stands for any defined parameter. We
+
+!listing htgr/generic-pbr-tutorial/step1.i block=Mesh
+
+The parameters of `GeneratedMeshGenerator` are explained [here](https://mooseframework.inl.gov/source/meshgenerators/GeneratedMeshGenerator.html).
+In two-dimensional geometry, the boundaries are named bottom (id=0), right (id=1), top (id=2), and left (id=3).
+The line `coord_type = RZ` sets the coordinate system to be axisymmetric (aka RZ). The symmetry axis
+defaults to the y-axis, which is what we want, so we do not need to set it explicitly.
+
+To visualize the geometry, you can run:
+
+!listing
+./pronghorn-opt -i step1.i --mesh-only
+
+This command produces the file `step1_in.e` that can be visualized using paraview.
+It is shown in [step1geom].
+
+!media generic-pbr-tutorial/step1_geometry.png
+        style=width:10%
+        id=step1geom
+        caption=Geometry for Step 1.
+
+### Setting the Fluid Properties
+
+Fluid properties are set by first defining a `HeliumFluidProperties` fluid properties object
+and second by creating a material that inserts the fluid properties into functors (if you do not know what a functor is then click [here](https://mooseframework.inl.gov/moose/syntax/Materials/)).
+
+The first task is performed by the following input syntax:
+
+!listing htgr/generic-pbr-tutorial/step1.i block=FluidProperties
+
+The second task is performed by the `GeneralFunctorFluidProps` object:
+
+!listing htgr/generic-pbr-tutorial/step1.i block=Materials
+
+This object takes pressure and temperature provided as variables to parameters
+`pressure` and `T_fluid`, respectively, and computes fluid properties, such as
+density and viscosity. Density and viscosity are named `rho` and `mu`.
+Note that `T_fluid` is set to a constant value in this input file because
+we do not solve the heat equation and therefore have no fluid temperature defined
+as a variable.
+
+This object also requires providing functors to the `porosity`, `speed`, and `characteristic_length`
+parameters.
+
+Porosity is defined as an auxiliary variable here:
+
+!listing htgr/generic-pbr-tutorial/step1.i block=AuxVariables
+
+and set equal to the `bed_porosity` parameter of 0.39.
+`speed` is a functor that is computed automatically by the Navier-Stokes FV action,
+and `characteristic_length` is only used for computing Reynolds numbers, which are not
+required in Step 1; therefore, we set `characteristic_length = 1` for now.
+
+### Setting up the Thermal-Hydraulics Problem
+
+The thermal-hydraulics problem is set up using the finite volume Navier-Stokes action.
+This action is discussed in detail [here](https://mooseframework.inl.gov/source/actions/NSFVAction.html).
+The corresponding block in the input file is:
+
+!listing htgr/generic-pbr-tutorial/step1.i block=Modules
```

Comment:
```suggestion
!listing htgr/generic-pbr-tutorial/step1.i block=NavierStokes
```

nesting everything in Modules is a deprecated syntax

Reply:
I tested the input file & searched for the new syntax, but it does not work for me. How is the block supposed to look like?

Reply:
I was wrong, syntax has not changed. it just looks like a deprecated syntax.